### PR TITLE
add a function to switch between left-mouse pan or left-mouse rotate

### PR DESCRIPTION
**Pull request by toloudis**
Problem
=======
Allow switching between pan mode or rotate mode.
Resolves https://aicsjira.corp.alleninstitute.org/browse/AGENTVIZ-1337

Solution
========
Add a function called setPanningMode(boolean) where true causes left-drag to have pan behavior, and false causes left-drag to rotate.
The function is to be called explicitly on a tool button click.

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
Can run the testbed viewer - there's a Toggle Pan/Rotate button.

Keyfiles (delete if not relevant):
-----------------------
Some minor changes to consolidate the button layout in the test viewer

### CODE REVIEW

File: src/simularium/VisGeometry.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ import {
     WebGLRendererParameters,
     Mesh,
     Quaternion,
+    MOUSE,
```
Comment (schoinh) [inline src/simularium/VisGeometry.ts:33]: Strange, I can't find an entry for `Mouse` or `MOUSE` in the threejs docs!

File: src/simularium/VisGeometry.ts [hunk 2]
```diff
@@ -477,6 +478,26 @@ class VisGeometry {
...
     }
 
+    public setPanningMode(pan: boolean): void {
+        if (!pan) {
+            this.controls.enablePan = false;
+            this.controls.enableRotate = true;
+            this.controls.mouseButtons = {
+                LEFT: MOUSE.ROTATE,
+                MIDDLE: MOUSE.DOLLY,
+                RIGHT: MOUSE.PAN,
```
Comment (schoinh) [inline src/simularium/VisGeometry.ts:488]: Whoa, we can do left = rotate and right = pan? This almost sounds like we can just have this and not bother with hot keys/buttons :P

**Review by schoinh — APPROVED** [review]

File: src/simularium/VisGeometry.ts (hunk 1 above)
Comment (toloudis) [reply src/simularium/VisGeometry.ts:33]: https://threejs.org/docs/?q=orbit#examples/en/controls/OrbitControls.keys

Comment (toloudis) [reply src/simularium/VisGeometry.ts:33]: https://threejs.org/docs/?q=orbit#examples/en/controls/OrbitControls.mouseButtons

**Review by meganrm — APPROVED** [review]

**Pull request merged**